src/api/plugins.js

```diff
@@ -165,23 +165,23 @@ module.exports = function buildJSONApi(gridsome) {
             result.data.data.edges.map(async ({ node: plugin }) => {
               // Clean up plugin object, create full log_url, add docs url
               const logoUrl = plugin.logo_url
-                ? path.join(baseurl, plugin.logo_url)
+                ? new URL(plugin.logo_url, baseurl)
```

Comment:
I think gridsome is pre-processing the logo images so these URLs are not valid. For example: https://deploy-preview-877--meltano-hub.netlify.app/assets/logos/extractors/stackexchange.png.

That seems unrelated to the base URL issues, though.

Reply:
Yeah you're right. #878 should resolve the issue